app/src/main/java/co/runloop/influencer/viewmodel/ContactsViewModel.java

```diff
@@ -0,0 +1,48 @@
+package co.runloop.influencer.viewmodel;
+
+import android.app.Application;
+import android.arch.lifecycle.AndroidViewModel;
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+import android.util.Log;
+
+import java.util.List;
+
+import co.runloop.influencer.data.ContactsProvider;
+import co.runloop.influencer.model.Contact;
+import co.runloop.influencer.utils.SimpleObserver;
+
+public class ContactsViewModel extends AndroidViewModel {
+
+    private MutableLiveData<List<Contact>> contacts;
+    private ContactsProvider provider;
+    private SimpleObserver observer;
+
+    public ContactsViewModel(Application app) {
+        super(app);
+        contacts = new MutableLiveData<>();
+        provider = new ContactsProvider(app);
```

Comment:
What a difference ?

Reply:
I totally agree that in this case there is no difference. In most common implementations the app and the application context will be the same object. But in theory, it could be different. 

It's more a matter of practice, and how we think about such things

 As a general idea, I think it would be better

Reply:
ok